nixos/modules/services/networking/softether.nix

```diff
@@ -104,17 +113,17 @@ in
     (mkIf (cfg.vpnbridge.enable) {
       systemd.services.vpnbridge = {
         description = "SoftEther VPN Bridge";
-        after = [ "softether-init.service" "network.target" ];
-        wants = [ "softether-init.service" ];
-        wantedBy = [ "multi-user.target" ];
+        after = [ "softether-init.service" ];
+        requires = [ "softether-init.service" ];
+        wantedBy = [ "network-interfaces.target" ];
```

Comment:
Replace `network-interfaces.target` with `network.target`: #18491
Please confirm that it works as you expect afterwards.

Reply:
@orivej Oops, thank you for reviewed that, I have replaced.